packages/client-core/src/user/components/UserMenu/menus/ProfileMenu.tsx

```diff
@@ -74,6 +74,7 @@ import styles from '../index.module.scss'
 const termsOfService = config.client.tosAddress ?? '/terms-of-service'
 
 const logger = multiLogger.child({ component: 'engine:ecs:ProfileMenu', modifier: clientContextParams })
+const USERNAME_MAX_LENGTH = 60
```

Comment:
is there a backend validation for it as well?
we should add it as well

Reply:
done ty